src/SearchResults.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResults = props => {
+  return (
+    <table class="table">
+      <thead class="thead-dark">
+        <tr>
+          <th scope="col">ID</th>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room ID</th>
+          <th scope="col">Check in date</th>
+          <th scope="col">Check out date</th>
+          <th scope="col">Nights</th>
```

Comment:
This is good, however there is a way to do this that would save you having to have a line of code for each column. If you know every booking in the table will have data for all of these columns you could generate them dynamically using a `.map` function with `Object.keys`. If this doesn't make sense please let me know :)